scripts/build_renewable_profiles.py

```diff
@@ -308,6 +308,7 @@ def get_hydro_capacity_annual_hydro_generation(
                     "bus": "first",
                 }
             )
+            .reset_index()
```

Comment:
drop index?

Reply:
Reset not drop because I need the index, it is the shape_id as a result of groupby